MapboxCoreNavigationTests/PassiveLocationDataSourceTests.swift

```diff
@@ -45,6 +45,9 @@ class PassiveLocationDataSourceTests: XCTestCase {
             self.locationUpdateExpectation = locationUpdateExpectation
         }
         
+        func passiveLocationDataSourceDidChangeAuthorization(_ dataSource: PassiveLocationDataSource) {
+        }
```

Comment:
#2579 conflicts with this change, hopefully not too severely.

Reply:
Looks like we’ll have to remove cbaacf72612dc861991bd3df036049732706a2a0 and reapply it in the release-v1.0 branch, because this class got completely removed in #2579.